onnx/backend/test/case/node/meansquareerror.py

```diff
@@ -0,0 +1,143 @@
+        # Define operator attributes.
+        reduction = 'none'
+        weights = [0.8, 0.9]
+
+        # Create operator.
+        node = onnx.helper.make_node('MeanSquaredError',
+                                     inputs=['R', 'T'],
+                                     outputs=['X'],
+                                     reduction=reduction
+                                     )
+
+        # Define operator inputs
+        r = np.array([1.2, 2.5], dtype=np.float32)
+        t = np.array([1.1, 2.6], dtype=np.float32)
+
+        # Compute Mean Square Error
+        l = np.square(r - t)
+        l = np.multiply(weights, l)
+        mse = l.mean(axis=0)
+
+        # Check results
+        expect(node, inputs=[r, t], outputs=[mse], name='test_mean_square_error_none_weights')
+
+
+    @staticmethod
+    def export_mean_square_error_sum():  # type: () -> None
+        # Define operator attributes.
+        reduction = 'sum'
+        shape = [2, 3]
+        axes = [1]
+        keepdims = 1
+
+        # Create operator.
+        node = onnx.helper.make_node('MeanSquaredError',
+                                     inputs=['R', 'T'],
+                                     outputs=['X'],
+                                     reduction=reduction
+                                     )
+
+        # Define operator inputs
+        r = np.array([[1.2, 2.5, 3.1], [1.3, 2.3, 3.4]], dtype=np.float32)
+        t = np.array([[1.1, 2.6, 3.2], [1.4, 2.2, 3.3]], dtype=np.float32)
+
+        # Compute Mean Square Error
+        l = np.square(r - t)
+        mse = np.sum(l, axis=tuple(axes), keepdims=keepdims == 1)
+
+        # Check results
+        expect(node, inputs=[r, t], outputs=[mse], name='test_mean_square_error_sum')
+
+
+    @staticmethod
+    def export_mean_square_error_mean():  # type: () -> None
+        # Define operator attributes.
+        reduction = 'mean'
+        shape = [2, 3]
+        axes = [1]
+        keepdims = 1
+
+        # Create operator.
+        node = onnx.helper.make_node('MeanSquaredError',
+                                     inputs=['R', 'T'],
+                                     outputs=['X'],
+                                     reduction=reduction
+                                     )
+
+        # Define operator inputs
+        r = np.array([[1.2, 2.5, 3.1], [1.3, 2.3, 3.4]], dtype=np.float32)
+        t = np.array([[1.1, 2.6, 3.2], [1.4, 2.2, 3.3]], dtype=np.float32)
+
+        # Compute Mean Square Error
+        l = np.square(r - t)
+        mse = np.mean(l, axis=tuple(axes), keepdims=keepdims == 1)
+
+        # Check results
+        expect(node, inputs=[r, t], outputs=[mse], name='test_mean_square_error_mean')
+
+
+    @staticmethod
+    def export_mean_square_error_mean_do_not_keepdims():  # type: () -> None
+        # Define operator attributes.
+        reduction = 'mean'
+        shape = [3, 2, 2]
+        axes = [1]
+        keepdims = 0
```

Comment:
The MSE op does not have any keepdims/axes attribute, so it should not be part of the tests.